civis/tests/test_io.py

```diff
@@ -145,6 +145,19 @@ def test_dataframe_to_civis(self, *mocks):
         assert result.state == 'succeeded'
 
     @patch(swagger_import_str, return_value=civis_api_spec)
+    def test_civis_multifile_unload(self, *mocks):
+        sql = "SELECT * FROM scratch.api_client_test_fixture"
+        result = civis.io.civis_multifile_unload(sql,
+                                                 database='redshift-general')
+        assert set(result.keys()) == {'files', 'query', 'header'}
```

Comment:
Because [CIVP-10192](https://github.com/civisanalytics/console/pull/8898) has been merged, the result.keys() should now equal `{'entries', 'query', 'header'}`